src/compiler/checker.ts

```diff
@@ -13698,13 +13722,33 @@ namespace ts {
 
         function checkExternalModuleExports(node: SourceFile | ModuleDeclaration) {
             let moduleSymbol = getSymbolOfNode(node);
-            let links = getSymbolLinks(moduleSymbol);
+            let links: SymbolLinks = getSymbolLinks(moduleSymbol);

```

Comment:
I don't. :3 I wrote it in to use go to definition on it, and forgot to remove it.